External/FEXCore/Source/Interface/IR/Passes/DeadContextStoreElimination.cpp

```diff
@@ -482,6 +487,7 @@ bool RCLSE::RedundantStoreLoadElimination(FEXCore::IR::IREmitter *IREmit) {
           // Happens when we store in to a location then store again
           IREmit->Remove(LastStoreNode);
           Changed = true;
+          VerboseLog("WaW");
```

Comment:
Might want to remove some of these verbose debug logs

Reply:
Removed them for now, will work on a nicer way to log these things in the future